README.md

```diff
@@ -1,3 +1,37 @@
 # RawToFigure
 
 Raw to figure describes echemdbs' approaches towards research data management (RDM).
+
+## Contribute
+
+If you want to modify rawtofigures' content itself, get a copy of the latest
+unreleased version of this repository:
+
+```sh
+git clone https://github.com/echemdb/unitpackage.git
+```
+
+Move to the directory and install the dependencies with, i.e., `conda` or `mamba`
```

Comment:
```suggestion
Move to the directory and install the dependencies with
```

I guess you meant `e.g.` here but anyway, just stating the command is probably easier.